set.mm

```diff
@@ -266031,114 +266222,16 @@ left ideal which is also a right ideal (or a left ideal over the opposite
 
 $(
 =-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=
-  Nonzero rings and zero rings
+  Nonzero rings and zero rings (continued)
```

Comment:
Why not moving the whole section? There are only 5 theorems left...

Reply:
The first theorem uses df-drng, the second uses df-subrg, and the last one uses df-fld, so they would be moved to those sections respectively. I vaguely decided that it might be more unified if these theorems were together, but they could be moved up, just not to the df-nzr section which is before df-drng, df-subrg, and df-fld.

Reply:
I guess I'd move to those three sections as described but it is a non-blocking comment as far as I'm concerned.